public/server/db/dttc.json

```
@@ -0,0 +1,3 @@
+{
+  "definition": "Don't text the cell"
```

If you read the README, you'll see that for an abbreviation to be accepted, the first letter of each word has to begin with a capital letter. Please follow the example, and correct the other abbreviations you want to submit.
```suggestion
  "definition": "Don't Text The Cell"
```